book.md

```diff
@@ -1808,17 +1813,19 @@ categorized per branch predictor component they target, are:
   can enable an attacker to derive the value of the secret key. These papers
   demonstrate deriving the secret key from implementations of specific
   cryptographic kernels. It can also be used to break ASLR\index{ASLR}.
-- BTB: SBPA [@Aciicmez2007], BranchShadow [@Lee2017]. These earlier attacks are
-  based on making a branch in the spy process alias in the BTB with a targeted
-  branch in the victim process. They use methods such as timing difference, last
-  branch records, instruction traces or performance counters to measure whether
-  the branch in the spy process caused a specific state change in the BTB.
-- RSB: Hyper-Channel [@Bulygin2008]. In this case, a spy process invokes $N$
-  calls to fill up the return stack predictor. Then it lets the victim process
-  execute. Then, the spy process can measure how many of its return stack
-  entries have been removed from the RSB, by measuring the number of $N$ returns
-  that get mis-predicted. If the number of calls in the victim process is
-  dependent on secret information, this could leak it.
+- Branch Target Predictor\index{Branch target predictor}: SBPA [@Aciicmez2007],
+  BranchShadow [@Lee2017]. These earlier attacks are based on making a branch in
+  the spy process alias in the BTB with a targeted branch in the victim process.
+  They use methods such as timing difference, last branch records, instruction
+  traces or performance counters to measure whether the branch in the spy
+  process caused a specific state change in the BTB.
+- Return address predictor\index{return address predictor}: Hyper-Channel
+  [@Bulygin2008]. In this case, a spy process invokes $N$ calls to fill up the
+  return stack predictor. Then it lets the victim process execute. Then, the spy
+  process can measure how many of its return stack entries have been removed
+  from the RSB, by measuring the number of $N$ returns that get mis-predicted.
+  If the number of calls in the victim process is dependent on secret
+  information, this could leak it.
 
```

Comment:
It may also be worth mentioning IBP (indirect branch predictor) used to predict indirect branches (e.g., C++ virtual calls).
// "indirect target array (ITA)" is another term, although AFAIK it's specific to AMD implementations.